library/spdm_requester_lib/libspdm_req_key_exchange.c

```diff
@@ -347,6 +347,7 @@ static libspdm_return_t libspdm_try_send_receive_key_exchange(
     spdm_request_size = message_size - transport_header_size -
                         spdm_context->local_context.capability.transport_tail_size;
 
+    LIBSPDM_ASSERT (spdm_request_size >= sizeof(libspdm_key_exchange_request_mine_t));
```

Comment:
This is NOT necessary. It could be smaller.

Reply:
Thanks. I have fixed the code.